### DISCUSSION TITLE
TDesign似乎不支持自动切换深浅色模式？

### DISCUSSION THREAD

**Discussion by ZCY-G** [Tencent/tdesign-react#875] [Ideas]
JS的监听浏览器主题似乎被弃用了？

或许深浅色分拆成两套css，利用css3的prefers-color-scheme也是个不错的选择。

例如这样：

```css
@import 'index.css';
@import 'index.dark.css' (prefers-color-scheme: dark);
```

**Comment by honkinglin** [comment]
主要是考虑到有些用户并不想自动切换功能，所以把这个功能交给用户自由决定。
可以 js 判断是夜间场景手动给 html 添加 `theme-mode="dark"`
